src/freenet/clients/http/staticfiles/themes/winterfacey/theme.css

```diff
@@ -737,10 +822,1154 @@ body[id*="Sone"] {
 #sone {
   padding-top: 1em;
   background-color: #fff;
-  box-shadow: 0 0 10px rgba(0, 0, 0, 0.5);
+  -webkit-box-shadow: 0 0 10px rgba(0, 0, 0, .5);
+    -moz-box-shadow: 0 0 10px rgba(0, 0, 0, .5);
+      box-shadow: 0 0 10px rgba(0, 0, 0, .5);
 }
 
 #page div#content {
   top: 0 !important;
   margin-top: 0;
 }
+
+
+/*** Hamburger ***/
+#hamburger-box {
+  position: relative;
+  z-index: 10;
+  float: right;
+  width: 30px;
+  height: 30px;
+  margin-right: 10px;
+  margin-top: 11px;
+  -webkit-transform: rotate(0deg);
+    -moz-transform: rotate(0deg);
+      -ms-transform: rotate(0deg);
+        -o-transform: rotate(0deg);
+          transform: rotate(0deg);
```

Comment:
why rotate by 0 degree?

Reply:
It's like from (from - to) meaning. Or it can be called init value. Again, like a good style for cross-browser compatibility (you can see examples of approximately this search "hamburger menu rotate example").

Reply:
ok